spark/src/main/scala/smile/data/SparkDataFrame.scala

```diff
@@ -45,53 +50,9 @@ case class SparkRowTuple(row: org.apache.spark.sql.Row, val schema: StructType)
   override def getString(i: Int): String = row.getString(i)
   override def getDate(i: Int): java.time.LocalDate = row.getDate(i).toLocalDate()
   override def getDateTime(i: Int): java.time.LocalDateTime = row.getTimestamp(i).toLocalDateTime()
+  override def getTime(i: Int): java.time.LocalTime = row.getTimestamp(i).toLocalDateTime().toLocalTime()
   override def getStruct(i: Int): SparkRowTuple = {
     val tuple = row.getStruct(i)
     SparkRowTuple(tuple, SparkDataTypes.smileSchema(tuple.schema))
   }
-}
-
-object SparkDataFrame {
-  /** Returns a local Smile DataFrame. */
-  def apply(df: org.apache.spark.sql.DataFrame): DataFrame = {
-    val schema = SparkDataTypes.smileSchema(df.schema)
-    DataFrame.of(
-      df.collect()
-        .map(row => SparkRowTuple(row, schema))
-        .toList
-        .asJava)
-  }
-}
-
-object SparkDataTypes {
```

Comment:
Why are these removed?

Reply:
I removed the implicits conversions because I thought it was not a good scala practice and it's probably never going to be used by users directly. Theses methods are still present here : spark/src/main/scala/org/apache/spark/smile/SparkDataTypes.scala as actual methods